cli/tests/026_workers.ts

```diff
@@ -1,26 +0,0 @@
-const jsWorker = new Worker("./subdir/test_worker.js", {
-  type: "module",
-  name: "jsWorker"
-});
-const tsWorker = new Worker("./subdir/test_worker.ts", {
-  type: "module",
-  name: "tsWorker"
-});
-
-tsWorker.onmessage = (e): void => {
-  console.log("Received ts: " + e.data);
-};
-
-jsWorker.onmessage = (e): void => {
-  console.log("Received js: " + e.data);
-
-  tsWorker.postMessage("Hello World");
-};
-
-jsWorker.onerror = (e: Event): void => {
-  e.preventDefault();
-  console.log("called onerror in script");
-  jsWorker.postMessage("Hello World");
-};
-
-jsWorker.postMessage("Hello World");
```

Comment:
Does this test pass tho? Maybe we should just leave it even though it's duplicated in the unit tests. It's a nice little stand alone example.

Reply:
Reverted